cartographer_toyota_hsr/configuration_files/toyota_hsr.lua

```diff
@@ -17,28 +17,29 @@ include "map_builder.lua"
 options = {
   map_builder = MAP_BUILDER,
   map_frame = "map",
-  tracking_frame = "gyro_link",
+  tracking_frame = "base_footprint",
   published_frame = "odom",
   odom_frame = "odom",
   provide_odom_frame = false,
-  use_odometry_data = true,
+  use_odometry_data = false,  -- TODO(macmason): Re-enable odometry once we enable the IMU.
   use_constant_odometry_variance = true,
   constant_odometry_translational_variance = 1e-7,
   constant_odometry_rotational_variance = 1e-7,
-  publish_occupancy_grid = false,
+  publish_occupancy_grid = true,
   use_horizontal_laser = true,
   use_horizontal_multi_echo_laser = false,
   horizontal_laser_min_range = 0.,
-  horizontal_laser_max_range = 30.,
+  horizontal_laser_max_range = 20.,
   horizontal_laser_missing_echo_ray_length = 5.,
   num_lasers_3d = 0,
-  lookup_transform_timeout_sec = 0.2,
+  lookup_transform_timeout_sec = 0.1,
   submap_publish_period_sec = 0.3,
   pose_publish_period_sec = 5e-3,
 }
 
-MAP_BUILDER.use_trajectory_builder_2d = true
-TRAJECTORY_BUILDER_2D.use_imu_data = true
-TRAJECTORY_BUILDER_2D.use_online_correlative_scan_matching = true
+options.map_builder.trajectory_builder_2d.submaps.resolution = 0.05
```

Comment:
The default resolution is 0.05 cm (as defined in cartographer/configuration_files/trajectory_builder_2d.lua).

Instead of writing "options.map_builder.trajectory_builder_2d", use the constant TRAJECTORY_BUILDER_2D which refers to the same dictionary.